db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180613231032) do
+ActiveRecord::Schema.define(version: 2015_10_18_191859) do
```

Comment:
Travis apparently reads this older date and thinks we have "pending migrations' and fails.

I recommend just using the same date (2018 06 13, 23:10:32) as before, with or without the underscores. (I don't get what this all means, to tell the truth, but I think it would make Travis pass the tests again.)